compatibility/support/shared_examples.rb

```diff
@@ -27,4 +27,15 @@
 
     expect(comparator.errors).to be_empty, "There were comparison errors: #{comparator.errors}"
   end
+
+  it 'ensures a consistent `testRunStartedId` across the entire test run' do
```

Comment:
@davidjgoss I think this removes the need to sanity check anything. Spoke with Rien on the call about this yesterday. My understanding here is every id should be identical across the suite for this check